modules/ROOT/pages/architecture/hexagonale_architecture.adoc

```diff
@@ -0,0 +1,218 @@
+:imagesdir: ../images
+// Open Questions/Discussion: 
+// - Is everything considered in the structure or are there things we cannot handle currently?
+// - What about shared technical elements? (AOP for logging across multiple rest adapters)
+// - How to organize multiple modules in a modular monolith and what about shared stuff there? 
+// - Should we really name the adapters after the protocol or more speaking (e.g. reservation for a inbound adapter for reservation) 
+// - Is structuring core and adapters in packages enough or should we use gradle modules or java modules?
+
+// Todos:
+// - Create ADR (Architecture Design records)
+// - Mention that the structure can be made more lean at any time. Consider three case:
+//   - Complex application with great logic (Probably JPA and entities different, also a great core layer)
+//   - Application containing logic, but entities and JPA do not differ (potentially a case where the entities in Core can contain JPA specific annotations t avoid mapping)
+//   - Pure CRUD application (Leave out core)
```

Comment:
Absolutely fine for CRUD to keep it simple, but that's not a hexagonale architecture for me anymore.

Reply:
True. Maybe a general architecture overview would help. And architecture styles for small applications (serverless / CRUD /...)

Reply:
https://github.com/devonfw/java/issues/42